scripts/o3de/o3de/cmake.py

```diff
@@ -208,6 +211,8 @@ def add_parser_args(parser):
                        help='The path to the project.')
     group.add_argument('-ep', '--engine-path', type=pathlib.Path, required=False,
                        help='The path to the engine.')
+    group.add_argument('-ed', '--external-subdirs', type=str, required=False,
+                       help='Additional list of subdirectories.')
```

Comment:
Can you also add a the `nargs='*'` option so this option can be used 0 or more times?